src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -93,6 +98,35 @@ def _create_temp_corpus_dir(self, name):
     engine_common.recreate_directory(new_corpus_directory)
     return new_corpus_directory
 
+  def save_corpus(self, source, destination):
+    """Saves syzkaller to folder so it is backed up to the cloud.
+
+    Args:
+      source: Folder where syzkaller corpus is.
+      destination: Folder where the corpus is synced with the cloud.
+    """
+    source_file = os.path.join(source, CORPUS_FILENAME)
+    shell.create_directory(destination)
+    target_file = os.path.join(destination,
+                               settings.get_device_codename() + '.db')
+    if os.path.isfile(source_file):
+      shutil.copy(source_file, target_file)
+
+  def init_corpus(self, source, destination):
```

Comment:
nit: s/source/source_dir/ and s/destination/destionation_dir/ to be clearer. Same for save_corpus.